linkedin_v2/linkedin.py

```diff
@@ -187,10 +197,243 @@ def get_profile(self, member_id=None, member_url=None, selectors=None,
+                                "text": title
+                            },
+                            "description": {
+                                "attributes": [],
+                                "text": description
+                            },
+                            "thumbnails": [],
+                            "status": "READY"
+                        }
+                    ],
+                    "shareCommentary": {
+                        "attributes": [],
+                        "text": ""
+                    }
+                }
+            },
+            "visibility": {
+                "com.linkedin.ugc.MemberNetworkVisibility": "CONTAINER"
+            }
+        }
+        if shareCommentary is not None:
+            post['specificContent']['shareCommentary'] = shareCommentary
+        response = self.make_request(
+            'POST', ENDPOINTS.SHARE, data=json.dumps(post))
+        raise_for_error(response)
+        return response.json()
+
+    def get_company_updates(self, organization_id, post):
+        url = '%s/people/id=%s/organizations/%s' % (
+            ENDPOINTS.SHARE, self.get_profile['id'], organization_id)
+        response = self.make_request(
+            'POST', url, data=json.dumps(post))
+        raise_for_error(response)
+        return response.json()
+
+    def get_group(self):
+        print(ENDPOINTS.BASE)
+        url = "%s/groupMemberships?q=member&member=urn:li:person:%s&membershipStatuses=List(MEMBER,OWNER)" % (ENDPOINTS.BASE, self.get_profile()['id'])
+        response = self.make_request('GET', url)
+        # raise_for_error(response)
+        return response.json()
+
+    def get_memberships(self):
+        pass
+
+    def submit_company_share(self, **kwargs):
+        params = kwargs.pop('params', True)
+        response = self.make_request(
+            'POST', ENDPOINTS.SHARE, data=json.dumps(params))
+        raise_for_error(response)
+        return response.json()
+
+    def _submit_share(self, comment=None, title=None, description=None,
+                      submitted_url=None, submitted_image_url=None,
+                      visibility_code='anyone'):
+        profile = self.get_profile()
+        post = {
+            'owner': 'urn:li:person:%s' % profile['id'],
+            "text": {
+                "text": description
+            },
+            "subject": title
+        }
+
+        if comment is not None:
+            post['comment'] = comment
+        # if title is not None and submitted_url is not None:
+        #     post['content'] = {
+        #         'title': title,
+        #         'submitted-url': submitted_url,
+        #         'description': description,
+        #     }
+        if submitted_image_url:
+            post['content']['contentEntities']['thumbnails']['resolvedUrl'] = submitted_image_url
+        response = self.make_request('POST', 'https://api.linkedin.com/v2/shares', data=json.dumps(post))
+        return response.json()
```

Comment:
Can you add a new line to the end of the file? See: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file